fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java

```diff
@@ -83,6 +113,7 @@ public void run() {
         Database database = op.get();
         modifyTblReplicaCount(database, StatisticConstants.STATISTIC_TBL_NAME);
         modifyTblReplicaCount(database, StatisticConstants.HISTOGRAM_TBL_NAME);
+        modifyTblReplicaCount(database, AuditLoaderPlugin.AUDIT_LOG_TABLE);
```

Comment:
Should we move `StatisticConstants.STATISTIC_INTERNAL_TABLE_REPLICA_NUM` to this file and rename it to `INTERNAL_TABLE_REPLICA_NUM`?

Reply:
I will try to refactor this later, there are some other things need to be done

Reply:
got it